app/styles/_navbar-alt.less

```diff
@@ -111,19 +113,16 @@
       background: @navbar-pf-alt-color;
     }
   }
-  .nav-item-iconic {
+  .nav .nav-item-iconic {
     // utiltity links
     cursor: pointer;
     line-height: 1;
-    max-height: (@navbar-pf-alt-height - @navbar-pf-alt-border-width); // to keep Firefox from oversizing icons
```

Comment:
Yep.  And just triple checked for good measure.  :)

![screen shot 2016-07-06 at 9 22 27 am](https://cloud.githubusercontent.com/assets/895728/16619225/43e83df6-435b-11e6-8566-8a05481a627e.PNG)